app/controllers/nonsipp/bondsdisposal/BondsDisposalListController.scala

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.nonsipp.bondsdisposal
+
+import cats.implicits.toTraverseOps
+import com.google.inject.Inject
+import config.Constants
+import config.Refined.Max5000
+import config.Refined.Max5000.enumerable
+import controllers.PSRController
+import controllers.actions.IdentifyAndRequireData
+import controllers.nonsipp.bondsdisposal.BondsDisposalListController._
+import forms.RadioListFormProvider
+import models.SchemeId.Srn
+import models.requests.DataRequest
+import models.{Mode, Money, Pagination, SchemeHoldBond}
+import navigation.Navigator
+import pages.nonsipp.bondsdisposal.BondsDisposalListPage
+import pages.nonsipp.unregulatedorconnectedbonds.{
+  CostOfBondsPage,
+  IncomeFromBondsPage,
+  NameOfBondsPage,
+  WhyDoesSchemeHoldBondsPage
+}
+import play.api.data.Form
+import play.api.i18n.MessagesApi
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents, Result}
+import utils.ListUtils._
+import viewmodels.DisplayMessage.{Message, ParagraphMessage}
+import viewmodels.implicits._
+import viewmodels.models.{FormPageViewModel, ListRadiosRow, ListRadiosViewModel, PaginatedViewModel}
+import views.html.ListRadiosView
+
+import javax.inject.Named
+
+class BondsDisposalListController @Inject()(
+  override val messagesApi: MessagesApi,
+  @Named("non-sipp") navigator: Navigator,
+  identifyAndRequireData: IdentifyAndRequireData,
+  val controllerComponents: MessagesControllerComponents,
+  view: ListRadiosView,
+  formProvider: RadioListFormProvider
+) extends PSRController {
+
+  val form: Form[Max5000] = BondsDisposalListController.form(formProvider)
+
+  def onPageLoad(srn: Srn, page: Int, mode: Mode): Action[AnyContent] = identifyAndRequireData(srn) {
+    implicit request =>
+      // todo swap when BondsCompleted has been added
```

Comment:
FYI @B-Savage there may be some cross-over with your ticket and bonds completed  page